File: Python/2-Programs-based-on-Numbers/39-quadrants-of-coordinates.py
num1 = int(input("Enter X coordinate value: "))
num2 = int(input("Enter Y coordinate value: "))

if (num1>0 and num2>0):
    print("First Quadrant.")
elif (num1<0 and num2>0):
    print("Second Quadrant.")
elif (num1<0 and num2<0):
    print("Third Quadrant.")
elif (num1>0 and num2<0):
    print("Fourth Quadrant.")
elif (num1==0 and num2==0):
    print("Origin Point.")
elif (num1==0):
    print("On Y-axis.")
elif (num2==0):
    print("On X-axis.")
else:
    print("Invalid Coordinates.")

File: Java/2-Programs-based-on-Numbers/39-quadrants-of-coordinates.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter X coordinate value:");
        int num1 = getInput.nextInt();

        System.out.println("Enter Y coordinate value:");
        int num2 = getInput.nextInt();

        if (num1>0 && num2>0) {
            System.out.println("First Quadrant.");
        } else if (num1<0 && num2>0) {
            System.out.println("Second Quadrant.");
        } else if (num1<0 && num2<0) {
            System.out.println("Third Quadrant.");
        } else if (num1>0 && num2<0) {
            System.out.println("Fourth Quadrant.");
        } else if (num1==0) {
            System.out.println("On Y-axis.");
        } else if (num2==0) {
            System.out.println("On X-axis.");
        } else {
            System.out.println("Invalid Coordinates.");
        }
        getInput.close();
    } 
}